### PR TITLE
Update template urls

### PR DESCRIPTION
**Pull request by lrapoport-cf**
This updates template urls following the move of templates into the workers-sdk monorepo (https://github.com/cloudflare/workers-sdk/pull/2498). 

This also fixes url updates made as part of https://github.com/cloudflare/cloudflare-docs/pull/7242.

### CODE REVIEW

File: content/d1/tutorials/build-a-comments-api/index.md [hunk 1]
```diff
@@ -201,4 +201,4 @@ Now, when you make requests to `/api/*`, Hono will automatically generate and ad
 
 ## Conclusion
 
-In this example, you built a comments API for powering a blog. To see the full source for this D1-powered comments API, you can visit [cloudflare/templates/worker-d1-api](https://github.com/cloudflare/templates/tree/main/worker-d1-api).
+In this example, you built a comments API for powering a blog. To see the full source for this D1-powered comments API, you can visit [cloudflare/workers-sdk/tree/main/templates/worker-d1-api](https://github.com/cloudflare/workers-sdk/tree/main/templates/worker-d1-api).
```
Comment (lrapoport-cf) [inline content/d1/tutorials/build-a-comments-api/index.md]: wasn't sure this should be changed as such -- would `[cloudflare/workers-sdk/templates/worker-d1-api]` be better?

File: content/workers/wrangler-legacy/commands.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ Default values indicated by {{<type>}}=value{{</type>}}.
   - The type of project; one of `webpack`, `javascript`, or `rust`.
 
 - `--site` {{<prop-meta>}}optional{{</prop-meta>}}
-  - When defined, the default `$TEMPLATE` value is changed to [`cloudflare/worker-sites-template`](https://github.com/cloudflare/worker-sites-template). This scaffolds a [Workers Site](/workers/platform/sites/) project.
+  - When defined, the default `$TEMPLATE` value is changed to [`cloudflare/workers-sdk/tree/main/templates/worker-sites`](https://github.com/cloudflare/workers-sdk/tree/main/templates/worker-sites). This scaffolds a [Workers Site](/workers/platform/sites/) project.
```
Comment (lrapoport-cf) [inline content/workers/wrangler-legacy/commands.md]: `[cloudflare/workers-sdk/templates/worker-sites]` instead?

File: content/workers/wrangler-legacy/commands.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Default values indicated by {{<type>}}=value{{</type>}}.
...
```
Comment (lrapoport-cf) [inline content/workers/wrangler-legacy/commands.md]: `[cloudflare/workers-sdk/templates/worker-sites]` instead?

File: content/workers/platform/sites/start-from-scratch.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ This guide shows how to quickly start a new Workers Sites project from scratch.
     The following example creates a project called `my-site`:
 
     ```sh
-    $ git clone --depth=1 --branch=wrangler2 https://github.com/cloudflare/worker-sites-template my-site
+    $ git clone --depth=1 --branch=wrangler2 https://github.com/cloudflare/workers-sdk/tree/main/templates/worker-sites my-site
```
Comment (lrapoport-cf) [inline content/workers/platform/sites/start-from-scratch.md]: this won't work yet -- need to either hold off or migrate the wrangler2 branch to the monorepo

File: content/workers/platform/languages.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ The Workers platform fully supports JavaScript. Cloudflare recommends using Java
 | Language   | Template                                                                                          |
 | ---------- | ------------------------------------------------------------------------------------------------- |
 | JavaScript | [cloudflare/worker-template](https://github.com/cloudflare/worker-template)                       |
-| TypeScript | [cloudflare/worker-typescript-template](https://github.com/cloudflare/worker-typescript-template) |
+| TypeScript | [cloudflare/workers-sdk/tree/main/templates/worker-typescript](https://github.com/cloudflare/workers-sdk/tree/main/templates/worker-typescript) |
```
Comment (lrapoport-cf) [inline content/workers/platform/languages.md]: `[cloudflare/workers-sdk/templates/worker-typescript]` instead?

File: content/workers/platform/languages.md [hunk 6]
```diff
@@ -38,7 +38,7 @@ The Workers platform fully supports JavaScript. Cloudflare recommends using Java
 | Language | Template                                                                                          |
 | -------- | ------------------------------------------------------------------------------------------------- |
 | Rust     | [cloudflare/rustwasm-worker-template](https://github.com/cloudflare/rustwasm-worker-template)     |
-| C        | [cloudflare/worker-emscripten-template](https://github.com/cloudflare/worker-emscripten-template) |
+| C        | [cloudflare/workers-sdk/tree/main/templates/worker-emscripten](https://github.com/cloudflare/workers-sdk/tree/main/templates/worker-emscripten) |
```
Comment (lrapoport-cf) [inline content/workers/platform/languages.md]: `[cloudflare/workers-sdk/templates/worker-emscripten]` instead?

**Review by JacobMGEvans — APPROVED** [review]
Going to these URLs work. Thanks for the PR 😄

File: content/d1/tutorials/build-a-comments-api/index.md (hunk 1 above)
Comment (JacobMGEvans) [reply content/d1/tutorials/build-a-comments-api/index.md]: I think they were ones I just missed.

Comment (lrapoport-cf) [reply content/d1/tutorials/build-a-comments-api/index.md]: changed

File: content/workers/wrangler-legacy/commands.md (hunk 2 above)
Comment (lrapoport-cf) [reply content/workers/wrangler-legacy/commands.md]: done

File: content/workers/wrangler-legacy/commands.md (hunk 3 above)
Comment (lrapoport-cf) [reply content/workers/wrangler-legacy/commands.md]: done

File: content/workers/platform/languages.md (hunk 5 above)
Comment (lrapoport-cf) [reply content/workers/platform/languages.md]: done

File: content/workers/platform/languages.md (hunk 6 above)
Comment (lrapoport-cf) [reply content/workers/platform/languages.md]: done

File: content/workers/platform/sites/start-from-scratch.md (hunk 4 above)
Comment (lrapoport-cf) [reply content/workers/platform/sites/start-from-scratch.md]: reverted the change. this should be updated once the wrangler2 branch is migrated to the monorepo -- captured in https://github.com/cloudflare/workers-sdk/issues/2719

Comment (JacobMGEvans) [reply content/workers/platform/sites/start-from-scratch.md]: We could just change the `--branch` to `main`. The templates repo doesn't have a `wrangler2` branch.

**Review by lauragift21 — APPROVED** [review]

File: content/d1/tutorials/build-a-comments-api/index.md (hunk 1 above)
Comment (Hamidvakil) [reply content/d1/tutorials/build-a-comments-api/index.md]: yt

**Pull request merged**